scripts/target.ps1

```diff
@@ -27,7 +27,7 @@ $TARGET_SPECS = (
         "Runtime"   = "osx-x64";
         "PkgType"   = "tar.bz2";
         "IsUnix"    = $True;
-        "TargetId" = "macos";
+        "TargetId" = "macos-x64";
```

Comment:
what about mac on x64?

Reply:
you mean arm64. It's been handled on another PR - it's below. 
In this PR we're just normalizing because we had macos arch-less and making some adjustments to upload/publish routine. I'm testing it against ravendb.net/downloads now. Once I confirm it can parse it successfully we can merge this one